api/build.gradle

```diff
@@ -34,6 +34,19 @@ sourceSets {
     }
 }
 
+configurations.all {
```

Comment:
Looks like this is a generic configuration required for all the parent & child modules, if we keep this in parent(ccpay-payment-app) only, will this not work for the child modules(/api, /api-contract, /model etc) as well ?

Reply:
As discussed & shown please keep the config in Root Build.gradle [all projects section]